src/components/Tabs/Tabs.tsx

```diff
@@ -77,13 +64,19 @@ const getElementsFromContentData = (data, useHash) => {
 
   data.map((tab, index) => {
     let tempPanel;
-    // For URL hash enabled tabs, we want to use the custom anchor component.
+    // For URL hash enabled tabs, we need to add a custom `onClick` to handle the URL hash.
     const tempTab = useHash ? (
-      <CustomAnchorTab key={index} hashKey={index}>
+      <Tab
+        fontSize={["0", null, "1"]}
+        key={index}
+        onClick={() => onClickHash(`tab${index + 1}`)}
```

Comment:
Not necessary to move on but this could be the optional check so `<Tab ...`  is only rendered once
```
<Tab ... onClick={useHash ? onClickHash(...) : null}>{tab.label}</Tab>
```